inc/saiwred.h

```diff
@@ -92,9 +106,6 @@ typedef enum _sai_wred_attr_t
     /** 0 ~ 15 [sai_uint8_t], Default 0*/
     SAI_WRED_ATTR_WEIGHT = 0x0000000c,
```

Comment:
Hi Itai, 

Please let me know, how to enable ECN in case ECN enable/disable is not supported by colour?

Regards
Ashok D